awx/main/models/credential/__init__.py

```diff
@@ -489,6 +488,12 @@ class Meta:
         default=False,
         editable=False
     )
+    native_source = models.CharField(
```

Comment:
Be it defined in the awx source through a `NativeCredentialType` or introduced with a plugin, this looks like it will always reference a namespace of some sort. Why not call this `namespace` ?